.vsts-ci/linux-daily.yml

```diff
@@ -18,17 +18,9 @@ pr:
   branches:
     include:
     - master
-    - release*
-    - feature*
   paths:
     include:
-    - '*'
-    exclude:
-    - tools/releaseBuild/*
-    - tools/releaseBuild/azureDevOps/templates/*
-    - /.vsts-ci/misc-analysis.yml
-    - /.github/ISSUE_TEMPLATE/*
-    - /.dependabot/config.yml
+    - .vsts-ci/linux-daily.yml
```

Comment:
Does this mean only PR that changes `linux-daily.yml` file will trigger the build?

Reply:
Yes, previously it was triggering for nothing.